ci/checks/style.sh

```diff
@@ -29,6 +29,10 @@ FLAKE_RETVAL=$?
 FLAKE_CYTHON=`flake8 --config=python/.flake8.cython`
 FLAKE_CYTHON_RETVAL=$?
 
+# Run mypy and get results/return code
+MYPY_CUDF=`mypy --config=python/cudf/setup.cfg python/cudf/cudf`
+MYPY_CUDF_RETVAL=$?
```

Comment:
Does this code actually cause mypy to run? It looks like it's just setting some ENV variables.

Reply:
I'm no bash expert but I believe:

* Anything in backticks is evaluated in bash
* `$?` is bash jargon for the "return value of the last command"

In general, I've just followed the conventions used for other checks in this file.

Reply:
It's running it and setting the output to the ENV variable

Reply:
TIL